concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+Marchese-1983-311-208	208	vent	960	WIND	wind	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-209	209	village	930	VILLAGE	town	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-210	210	un	1493	ONE	one	LES NUMERAUX
+Marchese-1983-311-211	211	deux	1498	TWO	two	LES NUMERAUX
+Marchese-1983-311-212	212	trois	492	THREE	three	LES NUMERAUX
+Marchese-1983-311-213	213	quatre	1500	FOUR	four	LES NUMERAUX
+Marchese-1983-311-214	214	cinq	493	FIVE	five	LES NUMERAUX
+Marchese-1983-311-215	215	six	1703	SIX	six	LES NUMERAUX
+Marchese-1983-311-216	216	sept	1704	SEVEN	seven	LES NUMERAUX
+Marchese-1983-311-217	217	huit	1705	EIGHT	eight	LES NUMERAUX
+Marchese-1983-311-218	218	neuf	1483	NINE	nine	LES NUMERAUX
+Marchese-1983-311-219	219	dix	1515	TEN	ten	LES NUMERAUX
+Marchese-1983-311-220	220	vingt	1710	TWENTY	twenty	LES NUMERAUX
+Marchese-1983-311-221	221	cent	1634	HUNDRED	hundred	LES NUMERAUX
+Marchese-1983-311-222	222	acheter	1869	BUY	to buy	LES VERBES
+Marchese-1983-311-223	223	accoucher	1195	GIVE BIRTH	to give birth	LES VERBES
+Marchese-1983-311-224	224	aiguiser	251	SHARPEN (SOMETHING)	to sharpen	LES VERBES
+Marchese-1983-311-225	225	aimer	923	LOVE	to love	LES VERBES
+Marchese-1983-311-226	226	aller	695	GO	to go	LES VERBES
+Marchese-1983-311-227	227	allumer	737	LIGHT (IGNITE)	to light up	LES VERBES
+Marchese-1983-311-228	228	appeler	1084	CALL	to call	LES VERBES
+Marchese-1983-311-229	229	apprendre	504	LEARN	to learn	LES VERBES
+Marchese-1983-311-230	230	arriver	1387	ARRIVE	to arrive	LES VERBES
+Marchese-1983-311-231	231	asseoir	1416	SIT	to sit	LES VERBES
+Marchese-1983-311-232	232	attacher	1917	TIE	to tie	LES VERBES
+Marchese-1983-311-233	233	attendre	1590	WAIT (FOR)	to wait	LES VERBES
+Marchese-1983-311-234	234	avaler	897	SWALLOW	to swallow	LES VERBES
+Marchese-1983-311-235	235	balayer	230	SWEEP	to sweep	LES VERBES
+Marchese-1983-311-236	236	bâtir	1840	BUILD	to build	LES VERBES
+Marchese-1983-311-237	237	battre	2133	STRIKE OR BEAT	to beat, to hit	LES VERBES
+Marchese-1983-311-238	238	(se) battre	1423	FIGHT	to fight	LES VERBES
+Marchese-1983-311-239	239	boire	1401	DRINK	to drink	LES VERBES
+Marchese-1983-311-240	240	bouillir	792	BOIL (OF LIQUID)	to boil	LES VERBES
+Marchese-1983-311-241	241	brûler	2102	BURN	to burn	LES VERBES
+Marchese-1983-311-242	242	cacher	602	HIDE (CONCEAL)	to hide	LES VERBES
+Marchese-1983-311-243	243	chanter	1261	SING	to sing	LES VERBES
+Marchese-1983-311-244	244	chercher	1468	LOOK FOR	to search	LES VERBES
+Marchese-1983-311-245	245	choisir	1750	CHOOSE	to choose	LES VERBES
+Marchese-1983-311-246	246	comprendre	1536	UNDERSTAND	to understand	LES VERBES
+Marchese-1983-311-247	247	compter	1420	COUNT	to count	LES VERBES
+Marchese-1983-311-248	248	coucher	2071	LAY (VERB)	to lay down	LES VERBES
+Marchese-1983-311-249	249	couler	2003	FLOW	to flow	LES VERBES
+Marchese-1983-311-250	250	couper	1432	CUT	to cut	LES VERBES
+Marchese-1983-311-251	251	creuser	1418	DIG	to dig	LES VERBES
+Marchese-1983-311-252	252	crier	715	SHOUT	to yell	LES VERBES
```

Comment:
it has two meanings: to cry (=weinen) and to shout (=schreien), But as the give "yell" as translation, we can leave it here.

Reply:
rubbish, what I said, it seems to only have the meaning "shout", don't know why I thought it would also mean "weinen". Sorry.